README.md

```diff
@@ -957,6 +957,7 @@ Table of Contents
    * [Yodiz](https://www.yodiz.com/) — Agile development and issue tracking. Free up to 3 users, unlimited projects.
    * [YouTrack](https://www.jetbrains.com/youtrack/buy/#edition=incloud) — Free hosted YouTrack (InCloud) for FOSS projects, private projects (free for 3 users). Includes time tracking and agile boards
    * [zenhub.com](https://www.zenhub.com) — The only project management solution inside GitHub. Free for public repos, OSS and nonprofit organizations
+   * [zentao.pm](https://www.zentao.pm) — Application Lifecycle management system includes issue tracking, free open source version available. 
```

Comment:
in order to qualify the free aspect of the service must be delivered as a service and not be self hosted, looks to me like the OSS version is only self hosted?